images/gatekeeper/README.md

```diff
@@ -0,0 +1,22 @@
+<!--monopod:start-->
+# gatekeeper
+| | |
+| - | - |
+| **Status** | stable |
+| **OCI Reference** | `cgr.dev/chainguard/gatekeeper` |
+| **Variants/Tags** | ![](https://storage.googleapis.com/chainguard-images-build-outputs/summary/gatekeeper.svg) |
+
+*[Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+Minimalist Wolfi-based Gatekeeper which is a policy controller for Kubernetes
```

Comment:
We need some usage docs, even if it just points to how to use it in a Helm a chart, or however it's normally run.

Reply:
Thanks @amouat, I've added some usage details and updated the cmd with help command.